test/models/o_auth_access_data_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class OAuthAccessDataTest < ActiveSupport::TestCase
```

Comment:
Could you please write a test here or delete the file? What do you think you would want to / be able to test here?

Reply:
I prefer to delete it for now